schedule/functional/extra_tests_in_textmode.yaml

```diff
@@ -0,0 +1,75 @@
+name:           extra_tests_in_textmode
+description:    >
+    Maintainer: slindomansilla.
+    Extra tests about software in Basesystem and Server Application modules
+schedule:
+    - boot/boot_to_desktop
+    - console/prepare_test_data
+    - console/consoletest_setup
+    - console/zypper_lr_validate
+    - console/zypper_ref
+    - console/zypper_info
+    - console/validate_packages_and_patterns
+    - console/zypper_extend
+    - console/check_os_release
+    - console/openvswitch
+    - console/pam
+    - console/sshd
+    - console/update_alternatives
+    - console/rpm
+    - console/slp
+    - console/aplay
+    - console/wavpack
+    - console/libvorbis
+    - console/command_not_found
+    - console/openssl_alpn
+    - console/autoyast_removed
+    - console/cron
+    - console/syslog
+    - console/mta
+    - console/yast2_lan_device_settings
+    - console/check_default_network_manager
+    - console/git
+    - console/cups
+    - console/java
+    - console/sqlite3
+    - console/gdb # should go to extra_tests_in_textmode_sdk
+    - console/sysctl
+    - console/sysstat
+    - console/curl_ipv6
+    - console/wget_ipv6
+    - console/ca_certificates_mozilla
+    - console/unzip
+    - console/machinery # should go to extra_tests_in_textmode_phub
+    - console/gpg
+    - console/rsync
+    - console/clamav
+    - console/shells
+    - console/sudo
+    - console/dstat
+    - x11/evolution/evolution_prepare_servers
+    - console/mutt
+    - console/supportutils
+    - console/mdadm
+    - console/journalctl
+    - console/quota
+    - console/vhostmd
+    - console/rpcbind
+    - sysauth/sssd # should go to extra_tests_in_textmode_phub
+    - console/timezone
+    - console/procps
+    - console/lshw
+    - console/kmod
+    - console/suse_module_tools
+    - console/zziplib
+    - console/firewalld
+    - console/aaa_base
+    - console/osinfo_db
+    - console/orphaned_packages_check
+    - console/zypper_ref
+    - console/docker # should go to extra_tests_in_textmode_cont
```

Comment:
containers*

Reply:
I am following our own abbreviations: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/registration.pm#L387

Reply:
ok, it should be contm if we decide to use abbreviation

Reply:
ok, makes sense... still... 

![ew](https://media.giphy.com/media/yHUiExmU40A4U/giphy.gif)